Form-Controls/index.html

```diff
@@ -1,28 +1,104 @@
+    </svg>
+    <h1>Choose your Mondrian Art T-shirt</h1>
   </header>
-  <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+    <input type="radio" id="com-a" name="mondrian-group" value="com-a" required />
+    <input type="radio" id="com-c" name="mondrian-group" value="con-c" />
+    <input type="radio" id="com-red-blue-yel" name="mondrian-group" value="com-red-blue-yel" />
+
+    <main>
+      <div class="col-1">
+        <label for="fname">First Name <span class="required">*</span></label>
+        <input type="text" id="fname" name="fname" placeholder="First Name" required minlength="2" />
+      </div>
+      <div class="col-1">
+        <label for="lname">Last Name <span class="required">*</span></label>
+        <input type="text" id="lname" name="lname" placeholder="Last Name" required minlength="2" />
+      </div>
+      <div class="col-2">
+        <label for="email">Email Address <span class="required">*</span></label>
+        <input type="text" id="email" name="email" placeholder="Email" pattern="[^ @]*@[^ @]*" required />
+      </div>
+
+      <div class="col-2">
+        <fieldset>
+          <legend>
+            Choose your Mondrian design <span class="required">*</span>
+          </legend>
+          <div class="input-set">
 
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
+            <label for="com-a"><img class="mondrian-img" src="images/mondrian_composition_a.jpeg"
+                alt="mondrian_composition_a" required />Composition A</label>
+          </div>
+          <div class="input-set">
+            <label for="com-c"><img class="mondrian-img" src="images/mondrian_composition_c.jpeg"
+                alt="mondrian_composition_c" />
+              Composition C </label>
+          </div>
+          <div class="input-set">
+            <label for="com-red-blue-yel"> <img class="mondrian-img" src="images/mondrian_red_blue_yellow.jpg"
+                alt="mondrian_red_blue_yellow" />
+              Composition with Red Blue and Yellow
+            </label>
+          </div>
+        </fieldset>
+      </div>
+
+      <div class="col-1">
+        <label for="size">Size <span class="required">*</span></label>
+        <select id="size" name="size" required>
+          <option value="" disabled selected>Select your size...</option>
+          <option value="x-small">X-Small</option>
+          <option value="small">Small</option>
+          <option value="medium">Medium</option>
+          <option value="large">Large</option>
+          <option value="x-large">X-Large</option>
+          <option value="xx-large">XX-Large</option>
+        </select>
+      </div>
+      <div class="col-1">
+        <label for="date">Delivery date: <span class="required">*</span></label>
+        <input type="date" id="date" name="date" min="2023-10-19" max="2023-11-16" required />
```

Comment:
This is a Good way of solving this problem. In real life, you'd probably use JavaScript to make the start and end dates dynamic but this is Good for this stage of the course.